package.json

```diff
@@ -34,6 +34,7 @@
     "graceful-fs": "^4.2.11",
     "ignore": "^5.2.4",
     "jszip": "^3.10.1",
+    "jszip-utils": "^0.1.0",
```

Comment:
remove this dep.

Reply:
removed it now.

Reply:
@niteshgupta90 the lockfile still references `jszip-utils`

running `yarn install` again should remove that (you don't have to edit `yarn.lock` manually).

There's also some merge conflicts that need to be resolved to be able to merge this PR.